File: snippets/csharp/System.Threading.Tasks/Task/Overview/run1.cs
// <Snippet6>
using System;
using System.Threading.Tasks;

public class Example
{
    public static async Task Main()
    {
        await Task.Run(() =>
        {
            // Just loop.
            int counter = 0;
            for (counter = 0; counter <= 1000000; counter++)
            { }
            Console.WriteLine("Finished {0} loop iterations", counter);
        });
    }
}
// This example displays the following output:
//     Finished 1000001 loop iterations
// </Snippet6>

File: snippets/visualbasic/VS_Snippets_CLR_System/system.threading.tasks.task/vb/run1.vb
' Visual Basic .NET Document
Option Strict On

' <Snippet6>

Module Example
    Public Async Sub Main()
        Await Task.Run(
            Sub()
                ' Just loop.
                Dim counter As Integer = 0
                For counter = 0 To 1000000
                Next
                Console.WriteLine("Finished {0} loop iterations", counter)
            End Sub)
    End Sub
End Module
' This example displays the following output:
'     Finished 1000001 loop iterations
' </Snippet6>
